.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/
 app/
 node_modules/
+package-lock.json
```

Comment:
this is generally not needed, as this repo uses pnpm. i guess you can use npm but it's a bit worrying lol

Reply:
I did end up using pnpm... it was left over from when I originally installed the project. I figured I would add it to gitignore so that if someone else made the same mistake they wouldn't have to worry about pushing a lock file.

Reply:
Now that we updated the build instructions, this is not needed anymore imo. Previous instructions was kind of confusing because it still mentioned npm when you were supposed to use pnpm. Its much clearer now so if anyone tries to commit a lock file other than pnpm's, that's a red flag.